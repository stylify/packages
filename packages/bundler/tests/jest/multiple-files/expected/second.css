.f{
	color: yellow
}
.o{
	color: purple
}
.e{
	content: 'second'
}
.g{
	content: 'Nette'
}
.j{
	content: 'Symfony'
}
.k{
	content: 'Vue'
}
.l{
	content: 'vue-script'
}
.n{
	content: 'Excluded'
}
.p{
	content: 'vuejs component'
}
.h{
	font-size: 24px
}
.i{
	font-size: 25px
}
.m{
	font-size: 26px
}
.q{
	font-size: 48px
}
