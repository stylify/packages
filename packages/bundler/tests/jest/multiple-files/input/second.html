<!--
	stylify-files
		second/nette.latte
		second/symfony.twig
		second/vue.vue
		/second/html.html
	/stylify-files

	stylify-pregenerate content:^second^ /stylify-pregenerate
-->

<div class="color:yellow"></div>

<code>
<!--
stylify-files
	ignored.md
/stylify-files
-->
</code>
