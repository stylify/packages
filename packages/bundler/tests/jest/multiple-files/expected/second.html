<!--
	stylify-files
		second/nette.latte
		second/symfony.twig
		second/vue.vue
		/second/html.html
	/stylify-files

	stylify-pregenerate content:^second^ /stylify-pregenerate
-->

<div class="f"></div>

<code>
<!--
stylify-files
	ignored.md
/stylify-files
-->
</code>
